.travis.yml

```diff
@@ -21,6 +21,8 @@ env:
   - IMG=s3-loader/0.6.0
   - IMG=elasticsearch-loader/0.10.2
   - IMG=iglu-server/0.4.0
+  - IMG=igluctl/0.5.0
+  - IMG=igluctl/0.6.0
```

Comment:
Do you need both 0.5.0 and 0.6.0? We usually leave only the latest one for testing